src/App.tsx

```diff
@@ -1,61 +1,133 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import { ToDo } from './types/ToDo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const todosInfo: ToDo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState<ToDo[]>(todosInfo);
+  const [user, setUser] = useState('');
+  const [title, setTitle] = useState('');
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  function handleChange(e: React.BaseSyntheticEvent) {
+    const { name, value } = e.target;
+
+    if (name === 'title') {
+      setTitle(value);
+    }
+
+    if (name === 'user') {
+      setUser(value);
+    }
+  }
```

Comment:
good for now, but for future - for different inputs - create separated handlers plz